_layouts/post.html

```diff
@@ -101,7 +101,7 @@
 
   {%if page.releaseColumn != false %}
   <td>
-    {% if diff > 0 %}{{r.latest}}{%else%}NA{%endif%}
+    {% if diff > 0 or r.eol == false %}{{r.latest}}{%else%}NA{%endif%}
```

Comment:
On my local machine this causes the release numbers to show, but they're not showing on Netlify. I'm not sure how to check if Netlify is running, erroring out or what. I'm sure there's a link staring me in the face but I can't find it; if you point me at it I'll continue debugging.